src/full_node/blockchain.py

```diff
@@ -527,6 +543,13 @@ async def receive_block(
 
         # Always immediately add the block to the database, after updating blockchain state
         await self.block_store.add_block(block)
+        if block.proof_of_time is not None:
```

Comment:
All blocks should have a PoT here, so instead we should do:
assert block.proof_of_time is not None